### DISCUSSION TITLE
How to show categories values including both expenses and income

### DISCUSSION THREAD

**Discussion by kaztela** [firefly-iii/firefly-iii#6036] [General]
For example, lets say I have a category for Amazon. I would like that category to include my costs as well as my refunds.
How can I do that?

**Comment by JC5** [comment]
Show where exactly?

**Reply by kaztela** [reply]
In the dashboard. The columns in categories show only expenses. How can they consider also refunds?

**Reply by JC5** [reply]
You can't, sorry.

**Reply by kaztela** [reply]
can this be configured maybe in the future?

**Comment by paulius-valiunas** [comment]
This would be a great feature. If I get charged twice and get a refund for one of the transactions, I don't want my category break down to look like I bought the thing twice (especially if it's expensive). It throws off the statistics. If I got a refund for something I didn't like and then bought a different item to replace it, I don't want it to look like I bought both things. If we go out with friends, I pay for everyone's lunch and they each send me what they owe as a separate transaction, I don't want it to look like I spent all that money on restaurants. All these situations make the category widget pretty much useless because it doesn't reflect the amount I actually spent in those categories.
